extensions/spring-xd-extension-http/src/main/java/org/springframework/integration/x/http/NettyHttpInboundChannelAdapter.java

```
@@ -198,11 +235,21 @@ protected void doStart() {
 			this.messageConverter = new NettyInboundMessageConverter(getMessageBuilderFactory());
 		}
 		executionHandler = new ExecutionHandler(executor);
-		bootstrap = new ServerBootstrap(new NioServerSocketChannelFactory(Executors.newCachedThreadPool(),
-				Executors.newCachedThreadPool()));
-		bootstrap.setOption("child.tcpNoDelay", true);
-		bootstrap.setPipelineFactory(new PipelineFactory());
-		bootstrap.bind(new InetSocketAddress(this.port));
+		/* begin whu */
+		try {
+			/* end whu */
+			bootstrap = new ServerBootstrap(new NioServerSocketChannelFactory(Executors.newCachedThreadPool(),
+					Executors.newCachedThreadPool()));
+			bootstrap.setOption("child.tcpNoDelay", true);
+			bootstrap.setPipelineFactory(new PipelineFactory());
+			bootstrap.bind(new InetSocketAddress(this.port));
+			/* begin whu */
+		}
+		catch (Exception e) {
+			/* bind exception -> server already created by another channel */
+			bootstrap = null;
```

This needs to be fatal - we can't just eat the exception.